components/formatter/IngressTarget.vue

```diff
@@ -36,6 +39,9 @@ export default {
     };
   },
   computed: {
+    workloads() {
+      return Object.values(WORKLOAD_TYPES).flatMap(type => this.$store.getters['cluster/all'](type));
+    },
```

Comment:
This looks like it's returning a "dead" array that's not going to update when the workloads come and go.

Reply:
I messaged you on slack about this.

It's possible that I would need to change this so that it's not a dead array but it's difficult to determine since the stores appear to be returning dead arrays. I've tested this with the namespace resource as well and creating/deleting namespaces doesn't update the list page so I don't think this is isolated.

Would you like to open another ticket to track the store issues and handle this separately?

In the meantime I converted the computed property to a method which sidesteps the issue in all cases I've identified except when deleting a workload that an ingress is targeting. The link will remain.

Reply:
I continued to do more digging and I ended up finding two issues which I resolved to handle this.

1. https://github.com/rancher/dashboard/pull/626/files#diff-db87951c51e96f9a7bdb79780bd523d7L21
  We need to set instead of assign the properties on the object to ensure they're reactive.
2. https://github.com/rancher/dashboard/pull/626/files#diff-1e61f2c77dee2237446ce2968fbb90d2R103
Vue wasn't watching/updating on the workloads computed property change because it was only sometimes used in the render method. I added this vestigial attribute which ensures it will be referenced in the render. If you have a better way to do this please let me know.

It shouldn't actually matter that I'm returning a dead array from this computed method because it always returns a new array. In the vuex documents there's examples of using things like filter which also return a new array.